docs/turbulence.md

```diff
@@ -0,0 +1,100 @@
+# Driven turbulence simulations
+
+The turbulence problem generator uses explicit inverse Fourier transformations (iFTs)
+on each meshblock in order to reduce communication during the iFT.
+Thus, it is only efficient if comparatively few modes are used (say < 100).
+
+Quite generally, driven turbulence simulations start from uniform initial conditions
+(uniform density and pressure, some initial magnetic field configuration in case of an
+MHD setup, and the fluid at rest) and reach a state of stationary, isotropic (or anisotropic
+depending on the strength of the background magnetic field) turbulence after one to few
+large eddy turnover times (again depending on the background magnetic field strength).
+The large eddy turnover time is usually defined as `T = L/U` with `L` being the scale
+of the largest eddies and `U` the root mean square Mach number in the stationary regime.
+
+The current implementation uses the following forcing spectrum
+`(k/k_peak)^2 * (2 - (k/k_peak)^2)`.
+Here, `k_peak` is the peak wavenumber of the forcing spectrum. It is related the scales of the largest eddies as
+`L = 1/k_f` given that a box size of 1 is currently assumed/hardcoded.
+
+## Problem setup
+
+An example parameter file can be found in `inputs/turbulence.in`.
+
+A typical setup contains the following blocks in the input file:
+
+```
+<job>
+problem_id = turbulence
+
+<problem/turbulence>
+rho0         = 1.0      # initial mean density
+p0           = 1.0      # initial mean pressure
+b0           = 0.01     # initial magnetic field strength
+b_config     = 0        # 0 - net flux; 1 - no net flux uniform B; 2 - non net flux sin B; 4 - field loop
+kpeak        = 2.0      # characteristic wavenumber
+corr_time    = 1.0      # autocorrelation time of the OU forcing process
+rseed        = 20190729 # random seed of the OU forcing process
+sol_weight   = 1.0      # solenoidal weight of the acceleration field
+accel_rms    = 0.5      # root mean square value of the acceleration field
+num_modes    = 30       # number of wavemodes
+
+<modes>
+k_1_0	= +2
+k_1_1	= -1
+k_1_2	= +0
+k_2_0	= +1
+...
+```
+
+Following parameters can be changed to control both the initial state 
+
+- `rho0` initial mean density
+- `p0` initial mean thermal pressure
+- `b0` initial mean magnetic field strength
+- `b_config`
+  - `0`: net flux case (uniform B_x)
```

Comment:
isn't it uniform B_z?  (same comments for 1 and 2)

Reply:
Just double checked. It is B_x, see https://github.com/parthenon-hpc-lab/athenapk/blob/7ea10bbd55c7353acf5d2f6350ddc1f3c9f515f1/src/pgen/turbulence.cpp#L273